go/client/cmd_git_gc.go

```diff
@@ -0,0 +1,110 @@
+package client
+
+import (
+	"errors"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+type CmdGitGC struct {
+	libkb.Contextified
+	repoName keybase1.GitRepoName
+	teamName keybase1.TeamName
+	force    bool
+}
+
+func newCmdGitGC(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "gc",
+		ArgumentHelp: "<repo name> [--team=<team name>]",
+		Usage:        "Run garbage collection on a personal or team git repository.",
+		Description:  "`keybase git gc reponame` will run garbage collection on  a personal git\n   repo. `keybase git create reponame --team=treehouse` will run garbage\n   collection on a team git repo for the `treehouse` team.  Garbage collection\n   cleans up unnecessary files and optimizes the local repo.",
```

Comment:
What does "optimizes the local repo" mean? Does this mean that my local clone of the Keybase repo gets cleaned up too? (It doesn't sound like this command needs to be run inside my local clone though.)

Reply:
Oops, that's a typo, it's not supposed to say "local".  I copied that from the `git-gc` man page.